scheds/rust/scx_layered/src/main.rs

```diff
@@ -986,6 +993,39 @@ struct Layer {
     allowed_cpus: Cpumask,
 }
 
+fn resolve_cpus_pct_range(
+    cpus_range: &Option<(usize, usize)>,
+    cpus_range_pct: &Option<(usize, usize)>,
+    max_cpus: usize,
+) -> Result<(usize, usize)> {
+    match cpus_range {
```

Comment:
Doing `match (cpus_range, cpus_range_pct)` will probably make the code simpler.

Reply:
never thought to try that, thx.